### ISSUE TITLE
Printed new line is weird

### ISSUE THREAD

**Issue by mobsenpai** [skim-rs/skim#565]
After selection an option the new line printed shows like this->

Before select: I pressed `ctrl + r` it started skim and its normal.
![image](https://github.com/lotabout/skim/assets/92603465/bf5eac5d-eb89-42a9-a88e-7efc33a3144a)

After select: I selected `cmatrix` from list and now `cmat` is beside my starship prompt.
![image](https://github.com/lotabout/skim/assets/92603465/57edb4cf-72b3-4a3a-a664-07299365e9f8)

```ocaml
OS: Nixos
Shell: Bash
Prompt: Starship
```

> I don't know what informative to give for this one. As I don't have any skim configuration in my bashrc. I just have a nixos option enabled 

```nix      
skim = {
        enable = true;
        enableBashIntegration = true;
      };
```

**Comment by mobsenpai** [comment]
Looks like the project is abandoned.

**Comment by linuxmobile** [comment]
> Looks like the project is abandoned.

Regarding the fact that it is abandoned. As long as it works I have no problem using it. However, I find that there are people who are maintaining a fork. You could try it!

https://github.com/nrdxp/skim/

**Regarding the error. In zsh I don't have this problem. I can't tell you why!**

**Comment by LoricAndre** [comment]
There's a good chance that this is due to your starship configuration rather than skim itself, could you please share it ?

**Comment by mobsenpai** [comment]
Okay. I am currently not on my pc so can only share my nix dotfiles.
[starship.nix](https://github.com/mobsenpai/hana/blob/cbf4def45e6db0e9c763c4c06c09258ea132b6c3/modules/home-manager/shell/starship.nix#L4)

**Comment by LoricAndre** [comment]
This doesn't really look like the config for the prompt above, could you please share your $PS1 and $PS2 ?
I'm looking for something like a `\r` somewhere that would justify this

**Comment by LoricAndre** [comment]
Thanks.
Just to be sure, does the issue happen only right after you exit the `ctrl-r` menu or does it persist after you press `enter` to get a new prompt line for example ?
Also, skim 0.10.4 is a bit outdated, could you please update ?

**Comment by mobsenpai** [comment]
Yeah I press enter and what I wrote before pressing `enter` stays there instead of yk completely being replaced with the chosen history. 
So I only have 0.10.4 version is all of nixos repositories and I would have to make some changes manually in my config to use the latest version

**Comment by LoricAndre** [comment]
I'm not sure there is much we can do until nix-pkgs updates skim to a later version unfortunately

**Comment by mobsenpai** [comment]
Basically its not clearing the terminal well. The search term persists and the selected option gets appended, unlike fzf where the search term is replaced entirely

**Comment by mobsenpai** [comment]
Please check the latest edit. I have added a video file.

@LoricAndre 
@linuxmobile

**Comment by linuxmobile** [comment]
> Please check the latest edit. I have added a video file.
> 
> [@LoricAndre](https://github.com/LoricAndre) [@linuxmobile](https://github.com/linuxmobile)

Holy, this issue is too long ago, what is wrong with the video, can you explain me please?

**Comment by mobsenpai** [comment]
@linuxmobile the search term shouldn't be there right, like it should get completely replaced with the item I have selected

**Comment by mobsenpai** [comment]
Seems like an update fixed it
